sdk/typescript/src/types/transactions.ts

```diff
@@ -104,6 +104,13 @@ export const SuiCallArg = union([
 		initialSharedVersion: string(),
 		mutable: boolean(),
 	}),
+	object({
```

Comment:
many of the superstruct types are in the process of being deprecated in the SDK, these changes need to be added here as well: https://github.com/MystenLabs/sui/blob/main/sdk/typescript/src/client/types/response/transactions.ts#L122

Reply:
Thanks for the heads up! Will rebase and make those changes here :)